source/OutfitterPanel.cpp

```diff
@@ -315,10 +343,26 @@ double OutfitterPanel::DrawDetails(const Point &center)
 
 
 
-ShopPanel::BuyResult OutfitterPanel::CanBuy(bool onlyOwned) const
+bool OutfitterPanel::IsInShop() const
```

Comment:
I'm not sure if this is a good name for this function given that it includes also items that the player has.

Reply:
Technically the player doesn't own the "stock" anymore, the shop does.  It's available because it's either in infinite Shop supply or limited Stock (still owned by the shop) supply.

We _are_ talking about `IsInShop`, right?

Reply:
ah yes nevermind